_studio/mfx_lib/decode/mjpeg/src/mfx_mjpeg_dec_decode.cpp

```diff
@@ -778,6 +778,11 @@ mfxStatus VideoDECODEMJPEG::DecodeFrameCheck(mfxBitstream *bs, mfxFrameSurface1
     mfxU32 numPic = 0;
     mfxU32 picToCollect = (MFX_PICSTRUCT_PROGRESSIVE == m_vPar.mfx.FrameInfo.PicStruct) ? 1 : 2;
 
+    //gpu session priority 
+    UMC::VideoDecoderParams* videoDecoderParams = &(decoder->umcVideoParams);
+    if (videoDecoderParams != nullptr && videoDecoderParams->pVideoAccelerator != nullptr)
```

Comment:
no sense to check this pointer, ```videoDecoderParams``` can't be null, since it is address of local variable